src/tools/install-git-hook/src/pre-commit.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# Call `tidy --bless` before each commit
+#
+# To enable this hook, run `./x.py run install-git-hook`.
+# To disable it, run `./x.py run install-git-hook --remove`
+set -Eeuo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel);
```

Comment:
This will break if there are spaces in the filename.
```suggestion
ROOT_DIR="$(git rev-parse --show-toplevel)";
```